### PR TITLE
Create instances

### PR DESCRIPTION
**Pull request by cheina97**
Components modified:

- TemplatesTableRow

Added the mutation function to the "Create" button, which allows to create a new instance.

### CODE REVIEW

File: frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx [hunk 1]
```diff
@@ -38,7 +46,7 @@ const TemplatesTableLogic: FC<ITemplateTableLogicProps> = ({ ...props }) => {
     error: errorInstances,
     startPolling: startPollingInstances,
   } = useOwnedInstancesQuery({
-    variables: { tenantNamespace: `tenant-${userId}` },
+    variables: { tenantNamespace: tenantNamespace },
```
Comment (QcFe) [inline frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx:49]:
```suggestion
    variables: { tenantNamespace },
```
You could avoid repeating the name and the value in objects when it's the same. There are several repetitions of this.

File: frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx [hunk 2]
```diff
@@ -32,31 +35,45 @@ const TemplatesTableRow: FC<ITemplatesTableRowProps> = ({ ...props }) => {
...
-              </Tooltip>
+        <div
+          className="flex w-full items-center cursor-pointer"
+          onClick={() =>
+            activeInstances
+              ? expandedId[0] === id
+                ? setExpandedId([''])
+                : setExpandedId([id])
+              : null
+          }
```
Comment (QcFe) [inline frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx:56]: I'd extract this into a function in order to improve readability.

**Review by QcFe — COMMENTED** [review]
Some minors style issues. I'll check the overall behavior in a while.

File: frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx (hunk 1 above)
Comment (cheina97) [reply frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx:49]: Changed it

File: frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx (hunk 2 above)
Comment (cheina97) [reply frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx]: I did it

**Review by GabriFila — APPROVED** [review]

**Pull request merged**